src/heads-up/style/tooltip.css

```diff
@@ -0,0 +1,96 @@
+dt {
+  cursor: pointer;
+}
+
+.tooltip {
+  display: block !important;
+  z-index: 10000;
+}
+
+.tooltip .tooltip-inner {
+  position: relative;
+  max-width: 350px;
+  font-size: .875rem;
+  background-color: var(--toolbar-bg-color);
+  color: var(--tab-selected-fg-color);
+  box-shadow: 0px 2px 2px 0 rgba(0, 0, 0, 0.2);
+  padding: 10px;
+}
+
+.tooltip[x-placement^="top"] .tooltip-inner::after {
```

Comment:
the pseudo elements share same styling (like content, width, height). Right now, there're quite some repetitions. Just group them together. Afterwards just specify the differences for each pseudo element.
```
.tooltip[x-placement^="top"] .tooltip-inner::after,
.tooltip[x-placement^="bottom"] .tooltip-inner::after,
.tooltip[x-placement^="bottom"] .tooltip-inner::after,
.tooltip[x-placement^="right"] .tooltip-inner::after {
...
}
```

Further, for readability, look how you're ordering the styles:
```
.class {
  content: '';
  position: absolute;
  bottom: -1em;
  left: 50%;
  width: 0;
  height: 0;
  margin-left: -0.5em;
  
  border: 0.5em solid black;
  border-color: transparent transparent var(--toolbar-bg-color) var(--toolbar-bg-color);
  box-shadow: -2px 2px 2px 0 rgba(0, 0, 0, 0.2);
 
  transform-origin: 0 0;
  transform: rotate(-45deg);
}
```